index.html

```diff
@@ -9,7 +9,13 @@
   </head>
   <body>
     <div id="calculator">
-      <div id="calculation_screen"></div>
+      <input class="input" type="text" />
+      <div class="col">
+        <button>C</button>
+        <button>%</button>
+        <button>M+</button>
```

Comment:
Are you going to implement the memory feature?  If not, you should handle this situation. Some ideas are removing it, displaying the button as disabled, or providing a tooltip on hover that shows that this 'feature is coming soon' etc